app/src/main/kotlin/org/koitharu/kotatsu/tracker/ui/feed/FeedViewModel.kt

```diff
@@ -85,7 +85,7 @@ class FeedViewModel @Inject constructor(
 		for (item in this) {
 			val date = timeAgo(item.createdAt)
 			if (prevDate != date) {
-				destination += ListHeader(date, 0, null)
+				destination += ListHeader(null, 0, date)
```

Comment:
Now user **must** pass both `text` and `textRes`, however this is not 100% correct too:
- If user pass not-null `text` and 0 `textRes`, `text` will be used - ok
- Both not-null `text` and non-zero `textRes` -- what result is expected?
- ...

Summarize, I suppose solution with multiple constructors is the best (except `sealed class`, but this solution is too expensive for such small class`), so class might be `data` with private primary constructor and some public secondary constructors